backend/tests_configs.ini

```diff
@@ -248,6 +248,23 @@ webdav.ui.enabled = False
 webdav.base_url = https://localhost:3030
 webdav.root_path = /webdav
 
+[functional_translation_test]
+app.enabled = contents/thread,contents/file,contents/html-document,contents/folder,upload_permission,share_content
+api.key = mysuperapikey
+preview.jpg.restricted_dims = True
+email.notification.activated = false
+website.base_url = http://localhost:6543
+user.reset_password.token_lifetime = 5
+frontend.serve = False
+email.notification.enabled_on_invitation = False
+webdav.ui.enabled = False
+webdav.base_url = https://localhost:3030
+webdav.root_path = /webdav
+translation_service.enabled = True
+translation_service.provider = systran
+translation_service.systran.api_url = https://systran_fake_server:5050
```

Comment:
Don't think it really and issue here. i explictly don't use extension to be sure it will not match now or later extension (and real domain name), the only "risk" which can happened is with custom dns server/ custom hostname file.

Reply:
but we effectively can use .invalid as it the standardized solution (don't know his existence until now) :https://fr.wikipedia.org/wiki/.invalid

Reply:
done.